### PR TITLE
Update create-an-org-wide-team.md

### PR DESCRIPTION
**Pull request by MarcoCarampanta**
Validated this info internally in the PR that implemented the feature.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/create-an-org-wide-team.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ ms.custom: seo-marvel-apr2020
...
 
-With org-wide teams, global admins can easily create a public team that pulls in every user in the organization and keeps the membership up to date with Active Directory as users join and leave the organization. Only global admins can create org-wide teams and currently an org-wide team is limited to organizations with no more than 5,000 users. There's also a limit of five org-wide teams per tenant. If these requirements are met, global admins will see **Org-wide** as an option when they select **Build a team from scratch** when creating a team. 
+With org-wide teams, global admins can easily create a public team that pulls in every user in the organization and keeps the membership up to date with Active Directory as users join and leave the organizaation. Only global admins can create org-wide teams and currently an org-wide team is limited to organizations with no more than 5,000 users. There's also a limit of five org-wide teams per tenant. If these requirements are met, global admins will see **Org-wide** as an option when they select **Build a team from scratch** when creating a team.
```
Comment (JohanFreelancer9) [inline Teams/create-an-org-wide-team.md:26]:
```suggestion
With org-wide teams, global admins can easily create a public team that pulls in every user in the organization and keeps the membership up to date with Active Directory as users join and leave the organization. Only global admins can create org-wide teams and currently, an org-wide team is limited to organizations with no more than 5,000 users. There's also a limit of five org-wide teams per tenant. If these requirements are met, global admins will see **Org-wide** as an option when they select **Build a team from scratch** when creating a team. 
```

Comment (JohanFreelancer9) [reply Teams/create-an-org-wide-team.md:26]: Please correct typo in the second occurrence of "organization". Also, insert a comma after "currently."

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**